### ISSUE TITLE
Global variables do not seem to carry over to loaded modules

### ISSUE THREAD

**Issue by yoshi1123** [tomblind/local-lua-debugger-vscode#58]
Reproduce example:

file1.lua:
```
g = "global"
require("file2.lua")
```

file2.lua
```
print(g)
```

Running `file.lua` with Lua will print "global", but running `file.lua` in the debugger will print `nil`.

**Comment by tomblind** [comment]
This looks like a bug with the interpreter mode's environment. Until it's fixed, you can work around it by running lua as a custom environment:
```
{
    "configurations": [
        {
            "name": "Debug Lua",
            "type": "lua-local",
            "request": "launch",
            "program": {
                "command": "lua",
            },
            "args": ["file1.lua"]
        }
    ]
}
```

**Comment by tomblind** [comment]
fixed by [079124e](https://github.com/tomblind/local-lua-debugger-vscode/commit/079124e826c04c2a94b8fe1791319504278397fe)
